ktor-utils/common/src/io/ktor/util/collections/MapDelegates.kt

```diff
@@ -78,6 +79,16 @@ public operator fun <T> SerializedMapValue<T>.setValue(thisRef: StringMap, prope
 public fun String.asBoolean(): SerializedMapValue<Boolean> =
     SerializedMapValue(this, Boolean::toString, String::toBoolean)
 
+
+/**
+ * Includes the attribute when the flag is set to true; else, excludes it.
+ *
+ * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.util.collections.asPresenceBoolean)
+ */
+@InternalAPI
+public fun String.asPresenceBoolean(): SerializedMapValue<Boolean?> =
```

Comment:
Probably in the future we could add a context parameter here. To reduce visibility